modules/java/native/java-grammar.ym

```diff
@@ -86,6 +87,14 @@ java_dest_custom_options
   |
   ;
 
+java_dest_empty_options
+  : java_dest_empty_option java_dest_empty_options
+  |
+  ;
+
+java_dest_empty_option
+  : LL_STRING { ;  }
+
```

Comment:
Just a note, bailing out with an error message would unfix the issue, because it would still force destinations to always be passed values for every possible optional parameter, behavior that might not be acceptable for some destinations. 

That being said, as an alternative to this patch, we could simply say that all destinations must assume all of their optional parameters will always get values, even if the values are defaults defined in the SCL blocks. I don't know if this would be limiting though.